### PR TITLE
Blog Post for the upcoming 2023 Board Member Elections

### PR DESCRIPTION
**Pull request by swatland610**
Just adds another blog post file with needed formatting

### CODE REVIEW

File: _posts/blog/2023-01-20-board-member-elections.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+## Running for the Board
+
+Are you a Member of Chi Hack Night and interested in running for the Board? Great! 
+
+Here’s what you need to do:
+ 1. **By Tuesday, January 31, 2023**
+    - If unsure, confirm that you (or the person you are nominating) are a Member of Chi Hack Night by reaching out to [membership@chihacknight.org](mailto:membership@chihacknight.org)
+
+2. **By Friday, February 4, 2023**
+ - Fill out the Candidate Statement Form. Make your case for why you should be part of the Chi Hack Night Board. All candidate statements will be shared out publicly ahead of the election.
```
Comment (lauriemerrell) [inline _posts/blog/2023-01-20-board-member-elections.md]: I think Candidate Statement Form is meant to be a link

File: _posts/blog/2023-01-20-board-member-elections.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+To vote in this election, or to run or be appointed to a Board seat, you must be a Member of Chi Hack Night as of Tuesday, January 31, 2023. If you are not sure of your membership status, you can email [membership@chihacknight.org](mailto:membership@chihacknight.org).
+
+## Running for the Board
+
+Are you a Member of Chi Hack Night and interested in running for the Board? Great! 
+
+Here’s what you need to do:
+ 1. **By Tuesday, January 31, 2023**
+    - If unsure, confirm that you (or the person you are nominating) are a Member of Chi Hack Night by reaching out to [membership@chihacknight.org](mailto:membership@chihacknight.org)
+
```
Comment (lauriemerrell) [inline _posts/blog/2023-01-20-board-member-elections.md:89]: Was there supposed to be a step here to fill out the nomination form?

File: _posts/blog/2023-01-20-board-member-elections.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+   </td>
+   <td>Steve Luker
+   </td>
+   <td>Pending Confirmation
+   </td>
+  </tr>
+</table>
+
+
+To vote in this election, or to run or be appointed to a Board seat, you must be a Member of Chi Hack Night as of Tuesday, January 31, 2023. If you are not sure of your membership status, you can email [membership@chihacknight.org](mailto:membership@chihacknight.org).
```
Comment (lauriemerrell) [inline _posts/blog/2023-01-20-board-member-elections.md]: Think the Google doc had a link describing membership

File: _posts/blog/2023-01-20-board-member-elections.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+author2: 
+author2_url:
+author2_image: 
+published: true
+featured: true
+---
+
+# Chi Hack Night’s Fourth Board Election
+
+On Tuesday, February 7, 2023, Chi Hack Night will hold our fourth annual member meeting and election for our Board of Directors. The Chi Hack Night Board of Directors, composed of 11 members, is responsible for governing our non-profit organization, as well as running our events, managing our website, keeping the books, and doing all the other things needed to make Chi Hack Night happen.
```
Comment (lauriemerrell) [inline _posts/blog/2023-01-20-board-member-elections.md]: Missing link for Board of Directors first mention

**Review by lauriemerrell — COMMENTED** [review]
Thanks Sean -- I think we are just missing some links

**Comment by swatland610** [comment]
Added in missing forms and final edits to 2023 election blog post.

**Pull request merged**